models/articles.model.js

```diff
@@ -40,4 +41,20 @@ exports.fetchArticles = () => {
         }
         return data.rows;
     })
+}
+
+exports.updateArticle = (article) => {
+    return fetchArticleById(article.article_id)
+    .then((data) => {
+        return db.query(`
+        UPDATE articles
+        SET votes = $1
+        WHERE article_id = $2
+        RETURNING *;
+    `, [data.votes + article.inc_votes, article.article_id])
```

Comment:
Ah i see you're doing this to get the current votes.
You can use SQL and do something like SET votes = votes + $1